src/App.js

```diff
@@ -4,15 +4,57 @@ import './App.css';
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <h1>
-          Eric F. Olsen
-        </h1>
-        <h3>
-          Software Engineer
-        </h3>
-      </header>
+      <Header />
+      {/* <Story />
+      <Resume />
+      <Projects />
+      <Testimonials /> */}
+    </div>
+  );
+}
+
+function Header() {
+  return (
+    <header className="App-header">
+      <img src={logo} className="App-logo" alt="logo" />
+      <h1>
+        Eric F. Olsen
+      </h1>
+      <h3>
+        Software Engineer
+      </h3>
+    </header>
+  );
+}
+
+function Story() {
+  return (
+    <div className="section">
```

Comment:
You can actually use a `<section></section>` element for this https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Reply:
If you really want to make your page accessible for the blind, etc., you can Google "Axe Dev Tools chrome extention" and install that, then run the scanner on your page.  If you want to install it, I can show you how it works, but it's pretty easy.  Again, optional, but at Procore I try to get everyone to think about accessibility while they are developing their code.

Reply:
Thanks @bobjohnbob I'll look into the section element and see if it fits what I'm planning here. This "section" class will be used for smoothscroll later to indicate what to scroll to.

@jesseolsen I do want to keep accessibility in mind, thanks for the tip!